s3tests_boto3/functional/test_sts.py

```diff
@@ -80,6 +82,19 @@ def put_user_policy(iam_client,username,policyname,policy_document):
         role_err = e.response['Code']
     return (role_err,role_response)
 
+def get_s3_client_using_iam_creds():
+    iam_access_key = get_iam_access_key()
+    iam_secret_key = get_iam_secret_key()
+
+    s3_client_iam_creds = boto3.client('s3',
+                              aws_access_key_id = iam_access_key,
+                              aws_secret_access_key = iam_secret_key,
+                              endpoint_url=default_endpoint,
```

Comment:
where is default_endpoint defined? I got an name 'default_endpoint' is not defined error while running your tests.